src/targets/slack.js

```diff
@@ -129,7 +129,11 @@ function getFailureDetails(suite) {
   for (let i = 0; i < cases.length; i++) {
     const test_case = cases[i];
     if (test_case.status === 'FAIL') {
-      text += `*Test*: ${test_case.name}\n*Error*: ${truncate(test_case.failure, 150)}\n\n`;
+      text += `*Test*: ${test_case.name}\n`;
+      if (test_case.failure) {
+        text += `*Error*: ${truncate(test_case.failure, 150)}\n`;
+      }
+      text += '\n';
```

Comment:
It is better to add `NA` in case the failure is undefined.
```suggestion
      text += `*Test*: ${test_case.name}\n*Error*: ${truncate(test_case.failure ?? 'NA', 150)}\n\n`;
```

Do you mind making similar changes in `teams` and `chat`?

Reply:
For my use case there can be a lot of failures at a time so the extra lines of `Error: N/A` are going to add a lot of noise. I'm not sure it adds value.

Reply:
@ASaiAnudeep ok, comments addressed and tests are passing.